runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -156,6 +180,25 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick tick)
+	{
+		if (currentRound != null && config.notifyCallChange() && currentRound.getRoundTime() >= 10)
```

Comment:
What is `roundTime`'s value when you start a round? This code looks like it would fire a notification immediately when starting a round if the clock shows `0:30` when you begin, which would not be ideal.